### ISSUE TITLE
[unifi] Binding should search for sid case insensitive

### ISSUE THREAD

**Issue by PascalTurbo** [openhab/openhab-addons#11964]
Setting up unifi binding could get very confusing when using Alias as sid selector.

For example the Alias of a device is set to "Christians iPhone". I would expect to set up thing like this:

```
Thing wirelessClient iphone_christian "Christians iPhone" [ cid="Christians iPhone", considerHome=180 ]
```

But this configuration will not be successfull because Unifi Controller returns downcase `christians iphone`:

```
 - UniFiClient{id: 'xxxxxxxxx', mac: 'xx:xx:xx:xx:xx:xx', ip: '192.168.2.113', hostname: 'iphonevhristian', alias: 'christians iphone', wired: false, blocked: false, device: UniFiDevice{mac: 'xx:xx:xx:xx:xx:xx', name: 'UAP Wohnzimmer', model: 'U7LT', site: UniFiSite{name: 'default', desc: 'Default'}}}
```

I could only identify this by using debug log of unifi binding. This is a bit user unfriendly. I think binding should simply search for cid case independent.

**Comment by jlaur** [comment]
@Hilbrand - maybe something to consider in context of #11959 if you are refactoring this part?

**Comment by Hilbrand** [comment]
Actually this should already be solved in #11959 and can be tested in with UniFi beta version in the marketplace.
